app/views/display_task/index.html.erb

```diff
@@ -21,3 +26,18 @@
 <div class="add_task_button">
   <%= button_to "Add a new task!", "/tasks/new" %>
 </div>
+
+<h1 class="title">People</h1>
+<table class="table_data">
+  <tr="table_headers">
+    <th>Person</th>
+    <th>Number of Uncompleted Tasks</th>
+  </tr>
+
+  <% @all_people.each_with_index do |person, index| %>
```

Comment:
Another question about whether or not you need to be using `each_with_index` or you could just be using `each`.